server/web_ui/admin/Cargo.toml

```diff
@@ -0,0 +1,70 @@
+[package]
+name = "kanidmd_web_ui_admin"
+description = "Kanidm Server Web UI - Admin Interface"
+documentation = "https://docs.rs/kanidm/latest/kanidm/"
+
+version = "1.1.0-rc.14-dev"
```

Comment:
I guess that wasm still can't inherit workspaces?

Reply:
Sorted, seems they can now